### DISCUSSION TITLE
Velocity Support

### DISCUSSION THREAD

**Discussion by demon2749** [Zrips/CMI#5134] [Feature Requests]
CMI has support for BungeeCord, and I was hoping those same features could be implemented for Velocity (https://velocitypowered.com/) in a manner similar to CMIBungee

**Comment by KarlPro123** [comment]
Good idea!

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions/2515

**Comment by mrfloris** [comment]
https://www.spigotmc.org/resources/cmi-298-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/update?update=521168

There is a mention in the changelog for velocity support in today's CMI release.
You can get the free cmiv jar file here: https://www.zrips.net/cmiv/
